src/main/java/org/georchestra/cas/MessagesConfiguration.java

```diff
@@ -0,0 +1,66 @@
+package org.georchestra.cas;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+
+@Component
+@Order(1)
+class MessagesConfiguration implements CommandLineRunner {
+
+    @Autowired GeorchestraConfiguration georchestraConfiguration;
+    public static final String FILE_PATH = "webapps/cas/WEB-INF/classes/messages.properties";
+    public static final String EXPIRE_PASS_KEY = "screen.expiredpass.message";
+    public static final String EXPIRE_PASS_VALUE = "Please <a href=\"%s\">change your password</a>.";
+    public static final String WARNING_PASS_KEY = "password.expiration.warning";
+    public static final String WARNING_PASS_VALUE = "Your password expires in {0} day(s). Please <a href=\"%s\">change your password</a> now.";
+    public static final String MUST_CHANGE_PASS_KEY = "screen.mustchangepass.message";
+    public static final String MUST_CHANGE_PASS_VALUE = "Please <a href=\"%s\">change your password</a>.";
```

Comment:
the mix between templating syntax (format string `%s` vs `{0}`) looks weird to me

Reply:
{0} is related to messages.property (defined there) and %s is used here to substitute goal